2-mandatory/4-people-i-know.js

```diff
@@ -378,13 +378,23 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+for(let myFriend of friends){
+   if(myFriend.age >= 35){
+     thirtyFiveOrOlder.push(myFriend)
+   }
+ }
 
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
 let powerNetEmails = [];
+for(let myFriend of friends){
+   if(myFriend.company == "POWERNET"){
```

Comment:
use `===` where possible over `==` as JS can be pretty weird when it comes to `==`